open-apparel-registry/open-apparel-registry
# Remove hard coded domain from mail.py

## Overview
When a user uploads a list to be processed, they are informed via email when processing is complete. There is a function in mail.py that builds the URL back to the processed list, and adds it to the body of the email.

Previously, the domains for staging and production were hard-coded based on environment. This PR replaces that by passing the public external domain from terraform through to mail.py via an environment variable.

Connects [#89 in the client repo](https://github.com/open-apparel-registry/open-apparel-registry-clients/issues/89)

## Testing Instructions
Once this branch is deployed to staging:
* Upload a list for processing
* When the email notification is received, confirm that the domain matches the currently configured `r53_public_hosted_zone` for staging, currently https://oshstaging.openapparel.org

## Checklist
- [ ] Temporarily push this PR branch to jenkins for deployment to staging
- [ ] Drop commit that deploys this PR branch to staging
